src/app/components/feed/CreateFeed.js

```diff
@@ -0,0 +1,218 @@
+import React from 'react';
+import { FormattedMessage, FormattedHTMLMessage } from 'react-intl';
+import Checkbox from '@material-ui/core/Checkbox';
+import ListIcon from '@material-ui/icons/List';
+import styles from './CreateFeed.module.css';
+import ExternalLink from '../ExternalLink';
+import Select from '../cds/inputs/Select';
+import SwitchComponent from '../cds/inputs/SwitchComponent';
+import TextArea from '../cds/inputs/TextArea';
+import TextField from '../cds/inputs/TextField';
+import SchoolIcon from '../../icons/school.svg';
+import CorporateFareIcon from '../../icons/corporate_fare.svg';
+import OpenSourceIcon from '../../icons/open_source.svg';
+
+/*
+  TODO
+  - Review with Brian
+    - License icons
+    - Verify proper casing
+      - Review copy? (license ... can do with your code)
+    - Borderless cards
+    - Shared feed overline title
+*/
+
+const LicenseOption = ({
+  icon,
+  title,
+  description,
+  url,
+}) => (
+  <div className={styles.licenseOption}>
```

Comment:
Can we set the icon and the text color to be `var(--textDisabled)` when the checkbox is unchecked and when checked the license title should be `var(--textPrimary)` and the description of the license should be `var(--textSecondary)`